### ISSUE TITLE
Extension manager window not showing correctly

### ISSUE THREAD

**Issue by aepedraza** [adobe/brackets#11073]
Hello, this is my first post in this project. I found a graphical issue in my Brackets's installation. When I open the **Extension Manager Window**, is shown as follows
![2015-05-09-111049_1024x600_scrot](https://cloud.githubusercontent.com/assets/10976526/7550531/1a97cefc-f63c-11e4-9fb2-5c188ad14de0.png)
which is not draggable by default.
I installed the [Draggable modals](https://github.com/dnbard/brackets-draggable-modals) plugin and I can drag the window but the result is as follows:
![2015-05-09-111531_1024x600_scrot](https://cloud.githubusercontent.com/assets/10976526/7550552/c0e31528-f63c-11e4-8ff5-fd63461f1263.png)
![2015-05-09-111742_1024x600_scrot](https://cloud.githubusercontent.com/assets/10976526/7550561/0a558c36-f63d-11e4-93bb-3a66f57155eb.png)
As you can see, there is still a weird thing with the window.
As I write this, I'm testing with other windows (about, update, etc) and those are *"cutted off"* (is that correct to say? Sorry, sometimes my english is limited) too, so it's a problem with all windows. You can figure out what I mean with the following:
![2015-05-09-113116_1024x600_scrot](https://cloud.githubusercontent.com/assets/10976526/7550609/0893418e-f63f-11e4-9456-d42299706d6b.png)

I have Brackets 1.2 on Linux Ubuntu 14.04.2 with LXDE as GUI. I didn't tried to update the version yet, I'll try this later and comment the result.
Thanks in advance for helping me out and hope this project keeps growing, it's a great editor.

**Comment by MarcelGerber** [comment]
I _think_ that's an issue with the Extract extension you have installed.
Please also try downloading the latest release - Release 1.3 - to see if it already got fixed (I honestly don't know right now).

**Comment by ryanstewart** [comment]
@MarcelGerber @aepedraza I just added a version of Extract to the registry that *should* fix this. You should see it as an update in your extension registry. Can you try installing that and letting me know?

**Comment by aepedraza** [comment]
I don't know why but now my Brackets installation loads without the extensions (even more: they seem to be uninstalled although they're in the `Brackets/extensions/user` folder) and the problem seems to be fixed now. Right now I'm having another issue, which is that I can't close the program (and seems to be related with [#4611](https://github.com/adobe/brackets/issues/4611))
All I did was download the v1.3 and tried to install it but got cancelled because my netbook was running very slowly. After that I opened Brackets and loaded as I described above.
I'll update to v1.3 to see what happens with the extensions, the closing and the current issue.
